src/researches/polish/functionWords.js

```diff
@@ -0,0 +1,380 @@
+	"właściwym", "właściwymi", "wysocy", "wysoka", "wysoką", "wysoki", "wysokich", "wysokie", "wysokiego", "wysokiej",
+	"wysokiemu", "wysokim", "wysokimi", "wyżsi", "wyższa", "wyższą", "wyższe", "wyższego", "wyższej", "wyższemu",
+	"wyższy", "wyższych", "wyższym", "wyższymi" ];
+
+let generalAdverbs = [ "blisko", "bliżej", "ciągle", "ciężko", "czasami", "czasem", "częściej", "często", "dalej",
+	"daleko", "dawniej", "dawno", "dobrze", "dopiero", "fajnie", "fajniej", "gorzej", "inaczej", "ładnie", "łatwiej", "łatwo",
+	"lepiej", "najbliżej", "najczęściej", "najdalej", "najdawniej", "najfajniej", "najgorzej", "najłatwiej", "najlepiej",
+	"najniżej", "najpóźniej", "najprościej", "najszybciej", "najtrudniej", "najwcześniej", "najwyżej", "naprawdę",
+	"niedaleko", "niedawno", "nisko", "niżej", "ostatnio", "pewno", "póżniej", "późno", "prawie", "prościej", "prosto",
+	"prostu", "szybciej", "szybko", "trochę", "trudniej", "trudno", "wcześnie", "wcześniej", "wolno", "wszędzie",
+	"wysoko", "wyżej", "zazwyczaj", "źle" ];
+
+let timeWords = [ "dni", "dnia", "dniach", "dniami", "dnie", "dzień", "dzisiaj", "godzin", "godzina", "godzinach",
+	"godzinami", "godzinę", "godziny", "jutro", "lata", "latach", "latami", "miesiąc", "miesiąca", "miesiącach",
+	"miesiącami", "miesiące", "miesiącem", "miesiącu", "miesięcy", "minut", "minuta", "minutach", "minutę", "minuty",
+	"pojutrze", "przedwczoraj", "rok", "rokiem", "roku", "sekund", "sekunda", "sekundach", "sekundę", "sekundy",
+	"tydzień", "tygodni", "tygodnia", "tygodniach", "tygodniami", "tygodnie", "tygodniu", "wczoraj" ];
+
+let vagueNouns = [ "chwila", "chwilą", "chwilach", "chwilami", "chwile", "chwilę", "chwili", "chwilom", "część", "części",
+	"częścią", "częściach", "częściami", "częściom", "momencie", "moment", "ogóle", "osób", "osoba", "osobą", "osobach", "osobami",
+	"osobę", "osobie", "osobom", "osoby", "powód", "powodach", "powodami", "powodem", "powodom", "powodów", "powodowi",
+	"powodu", "powody", "powodzie", "przypadkiem", "przypadku", "raz", "razach", "razami", "razem", "razie", "razom",
+	"razów", "razowi", "razu", "razy", "rodzaj", "rodzajach", "rodzajami", "rodzajem", "rodzajom", "rodzajów",
+	"rodzajowi", "rodzaju", "rzecz", "rzeczą", "rzeczach", "rzeczami", "rzeczom", "rzeczy", "sposób", "sposobem",
+	"sprawa", "sprawą", "sprawach", "sprawami", "sprawę", "sprawie", "sprawom", "sprawy", "temacie", "temat", "tematach",
+	"tematami", "tematem", "tematom", "tematów", "tematowi", "tematu", "tematy" ];
+
+let titles = [ "dr", "dyr", "mgr", "p", "pan", "pani", "panie", "panowie", "prof" ];
+
+let interjections = [ "a", "ach", "aha", "aj", "akurat", "ał", "aua", "auć", "ba", "brawo", "e", "ech", "ehe", "ehm", "ej",
+	"ejże", "ekhm", "ekstra", "jej", "jejku", "łał", "och", "oh", "oho", "oj", "ojej", "ojejku", "phi", "precz", "super",
+	"uwaga", "wow" ];
+
+
+let recipeWords = [ "c", "cl", "cm", "dag", "deka", "dl", "g", "gram", "gramów", "gramy", "kg", "kilo", "l", "litr",
```

Comment:
Things like "mm", "m", "ft", and other measurement words still appear in the list of word combinations. Please consider adding them to the list of function words.

Reply:
done!